pending/xnviewmp.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    BleachBit
+    Copyright (C) 2008-2018 Andrew Ziem
+    https://www.bleachbit.org
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	CleanerML written by https://github.com/Tobias-B-Besemer
+	v0.1 done on 2018-12-19
+
+-->
+<cleaner id="xnviewmp">
+  <label>XnView MP for Windows</label>
+  <description>Picture Viewer</description>
+  <option id="customPath">
+    <label>Custom Path of XnView MP</label>
+    <description>Delete the custom path of XnView MP</description>
+        <action command="ini" search="glob" path="$APPDATA\XnViewMP\xnview.ini" name="customPath"/>
+        <action command="ini" search="glob" path="%ProgramFiles%\XnViewMP\xnview.ini" name="customPath"/>
+        <action command="ini" search="glob" path="%ProgramFiles(x86)%\XnViewMP\xnview.ini" name="customPath"/>
+        <action command="winreg" path="HKCU\Software\XnView\XnViewMP\Start\customPath" />
+  </option>
+  <option id="savefolder">
```

Comment:
```suggestion
  <option id="save_folder">
```

Reply:
I really don't saw that! I had chosen "savefolder" because by command="ini" "name" was "savefolder", too.

Reply:
OK, that's fine. Don't worry about it.